app/src/main/java/com/cornellappdev/volume/PublicationProfileActivity.kt

```diff
@@ -143,40 +114,40 @@ class PublicationProfileActivity : AppCompatActivity() {
                     }
                 }
                 if (publication.websiteURL.isNotEmpty()) {
-                    profile_link.text = publication.websiteURL
-                    profile_link_holder.setOnClickListener {
+                    binding.tvWebsiteLink.text = publication.websiteURL
+                    binding.clWebsiteHolder.setOnClickListener {
                         startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(publication.websiteURL)))
                     }
                 } else {
-                    profile_link_holder.visibility = View.GONE
+                    binding.clWebsiteHolder.visibility = View.GONE
                 }
                 if (instaURL.isNotEmpty()) {
                     setUpInstaOnClick(instaURL)
                 } else {
-                    findViewById<ConstraintLayout>(R.id.insta_holder).visibility = View.GONE
+                    binding.clInstaHolder.visibility = View.GONE
                 }
                 if (facebookURL.isNotEmpty()) {
-                    findViewById<ConstraintLayout>(R.id.fb_holder).setOnClickListener {
+                    binding.clFbHolder.setOnClickListener {
                         val intent = Intent(Intent.ACTION_VIEW, Uri.parse(facebookURL))
                         startActivity(intent)
                     }
                 } else {
-                    findViewById<ConstraintLayout>(R.id.fb_holder).visibility = View.GONE
+                    binding.clFbHolder.visibility = View.GONE
                 }
-                profile_name.text = publication.name
-                profile_shoutouts.text = publication.shoutouts.toInt().toString() + " shoutouts"
-                profile_desc.text = publication.bio
-                Picasso.get().load(publication.backgroundImageURL).into(profile_banner)
-                Picasso.get().load(publication.profileImageURL).into(profile_logo)
+                binding.tvName.text = publication.name
+                binding.tvShoutoutCount.text =
+                        publication.shoutouts.toInt().toString() + " shoutouts"
```

Comment:
In a future PR, replace these with format strings from the .xml file.

Reply:
yup soon to come